frontend/packages/integration-tests-cypress/tests/i18n/pseudolocalization.spec.ts

```diff
@@ -0,0 +1,30 @@
+import { checkErrors } from '../../support';
+import { timestamp } from '../../views/timestamp';
+import { shouldMatchRegex } from '../../views/common';
+
+describe('Localization', () => {
+  before(() => {
+    cy.login();
+  });
+
+  afterEach(() => {
+    checkErrors();
+  });
+
+  after(() => {
+    cy.logout();
+  });
+
+  it('pseudolocalizes navigation', () => {
+    cy.log('test navigation');
+    cy.visit('/dashboards?pseudolocalization=true&lng=en');
+    shouldMatchRegex('[data-cy=nav]', /\[[^a-zA-Z]+\]/);
```

Comment:
Maybe should add `regex` to  `cy.byTestID(id: string, regex:any)` and put the regex handling in `cy.byTestID` if regex param detected.  We should be using `data-test` per [our best practices](https://github.com/openshift/console/blob/master/frontend/packages/integration-tests-cypress/README.md#migrating-protractor-tests-to-cypress) which lends itself to `cy.byTestID`.